LSP-intelephense.sublime-settings

```diff
@@ -12,6 +14,7 @@
 			"feature_selector": "source.php",
 		},
 	],
+	"auto_complete_selector": "punctuation.accessor.arrow | punctuation.accessor.double-colon | punctuation.definition.variable | punctuation.separator.namespace | comment.block | punctuation.definition.tag.begin | punctuation.accessor.js",
```

Comment:
I guess we can simply use `punctuation.accessor` instead of `punctuation.accessor.arrow | punctuation.accessor.double-colon | punctuation.accessor.js`.